manifests/runner.pp

```diff
@@ -132,7 +132,9 @@
     $ssh_password = undef,
     $require = [ Class['gitlab_ci_multi_runner'] ],
 ) {
-    $description = $name
+    # Single Quote Description so that none of the commands that use
+    # it break on spaces
+    $description = "'${name}'"
```

Comment:
A valid concern - Turns out Puppet has a function for this - "shellquote". A bit of quick testing shows that it properly handles spaces, single quotes, and double quotes in shell commands. I've amended my commit to use it instead.